openedx/features/edly/tests/test_models.py

```diff
@@ -73,5 +79,39 @@ def test_edly_sub_organization_post_update_receiver(self):
         )
         edly_sub_organization_data = EdlySubOrganization.objects.filter(studio_site=edly_sub_organization.studio_site)
 
-        assert len(edly_sub_organization_data) == 1
+        assert edly_sub_organization_data.count() == 1
         assert edly_sub_organization_data[0].edly_organization.name == edly_organization.name
+
+
+class EdlyUserProfileTests(TestCase):
+    """
+    Tests for EdlyUserProfile Model.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        super(EdlyUserProfileTests, cls).setUpClass()
+        cls.user = UserFactory(password="test")
+
+    def test_edly_user_profile_post_save_receiver(self):
+        """
+        Test EdlyUserProfile model object creation.
```

Comment:
nit: name of methods should be in double quotes `"` or `

Reply:
Done